prep-css-display/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Display Properties</title>
+    <style>
+      div {
+        border: 4px solid blue;
+      }
+      span {
+        border: 4px solid green;
+        height: 100px;
+        width 100px;
+      }
+      button {
+        border: 4px solid red;
+        height: 40px;
+        width: 200px;
+      }
+      ul {
+        border: 4px solid purple;
+      }
+      li {
+        border: 4px solid magenta;
+        display: inline;
```

Comment:
Oop! There should be one more curly brace here at the end of your last rule set `}` 😄